CHANGELOG.md

```diff
@@ -58,6 +58,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * avifenc: Change the default value of the --jobs option from 1 to "all".
 * Update avifCropRectConvertCleanApertureBox() to the revised requirements in
   ISO/IEC 23000-22:2019/Amd. 2:2021 Section 7.3.6.7.
+* Fix memory errors reported in crbug.com/1501766 and crbug.com/1501770.
```

Comment:
Note that I repeated the v1.0.2 bug fixes in the Changed section of the next release from the main branch, because that release will diff against the v1.0.0 release.